mappings/cb359dba9731ed31a1c6baa8b769feb5578ea326aa0b005b9c7369a74f4e45.json

```diff
@@ -0,0 +1,63 @@
+{
+    "subject": "cb359dba9731ed31a1c6baa8b769feb5578ea326aa0b005b9c7369a74f4e45",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://twitter.com/theonlyoneada",
```

Comment:
Hi @HatMeme,

can you please add policyId to the description of the account so we can verify that?

Thanks!